content/docs/getting-started/getting-started-for-admins/index.md

```diff
@@ -35,10 +35,18 @@ If you wish to tightly restrict the access that Roadie has, please [follow this
 
 ### Step 1. Create a Personal Access Token (PAT)
 
-Visit the [developer settings of your GitHub account](https://github.com/settings/tokens) and create a PAT which has the scopes `repo`, `user`, `read:org` and `read:discussion` .
+Visit the [developer settings of your GitHub account](https://github.com/settings/tokens) and create a PAT which has the scopes `repo`, `user`, `read:org` and `read:discussion`.
 
 The official GitHub docs for creating a Personal Access Token are [here](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token).
 
+---
+
+**NOTE**
+
+If you will be using this PAT to collect organization teams and memebers, then the token's owner must be a member of the organization.
```

Comment:
I would explain what the owner cannot be, since some people won't know what outside collaborators are or that they exist in the first place. I would also make the assumption that they will use the PAT to collect org teams and members so we can simplify the logic.

```suggestion
⚠️ The token's owner must be a member of the organization, rather than an outside collaborator or other relation.
```